existing-communities/README.md

```diff
@@ -1 +1,47 @@
+Virtual communities have four necessary characteristics: interactivity, communicators, a publicly shared mediated (virtual) communication place and a minimum level of sustained membership{{"jones1997virtual" | cite}}. This means that in a virtual community, there must be a longlasting, selfsustaining communication *(interactivity)* about subjects (related to the relating characteristics) by two or more persons *(communicators)* over one ore more platform like a website or a chat-group *(communication place)* where there is a certain amount of active members *(sustained membership)*.
+## Existing Communities
+Before we can compare different communities and their structure, we first need to define a few (types of) communities. We will look at two types of real-life communities and two types of virtual communities, such that we can discus their similarities and differences.
+### Ecosystems
+Ecosystems can be seen as a natural form of a community. It is bound to a location where a diversity of animals and species interact with eachother{{"EcosysDef" | cite}}. Although there is no real communication, the combination of individual self-interest (the survival of their offspring or themself) creates a network of dependencies and interactions that one can interpret as a community.  
+A key factor of an ecosystem is stability. In a stable ecosystem{{"EcosysSDef" | cite}}, each species acts in such a way that they, sometimes inderectly, enable or do not interfere with the survival of another species. An ecosystem will remain stable as long as the system is able to deal with outside disturbances.
+
+### Intentional Communities
+Intentional communities are man-made communities, where its members attempt to realise a comen alternative way of life outside mainstream society{{"meijering2007intentional" | cite}}. Although there are no one list of criteria to characterise them, the most cited criteria are:
+
+1. No bonds by familial relationships only.
+2. A minimum of three to five adult members.
+3. Members join volunatarily
+4. Geographical and psychological seperation from mainstream society.
+5. A common ideology that is adhered to by all members.
+6. Sharing of (a part of) one's property.
+7. The interest of the group prevails over interests.
+
+Not all intentional communities fulfill all of these criteria, as we will see later on, but they give a good idea of what is and what is not a intentional community. Intentional communities can be quiet diverse. To get a better understanding of the different ways an intentional community can work, they can be split into four groups: ecological, communal, religious and practical communities{{"meijering2007intentional" | cite}}.
```

Comment:
`fulfill all of these criteria, as we will see later on, but they give a good idea` ==> `fulfill all of these criteria, but they give a good idea`

> Intentional communities can be **quite** diverse.

I would change the last sentence to: `Intentional communities can be split into four groups to get a better understanding of how they work: ecological etc..`